build.sbt

```diff
@@ -37,7 +38,7 @@ lazy val commonSettings: Project => Project = {
       Wart.IsInstanceOf,
       Wart.JavaConversions,
       //      Wart.JavaSerializable,
-      Wart.MutableDataStructures,
+      //      Wart.MutableDataStructures,
```

Comment:
We can suppress warts for typo-generated sources, see (https://www.wartremover.org/doc/install-setup.html).

It seems this is the way `wartremoverExcluded += baseDirectory.value / "src" / "main" / "scala" / "SomeFile.scala"`